tests/cypress/cypress/e2e/specsACMS/acms.content.spec.js

```diff
@@ -0,0 +1,170 @@
+import 'cypress-iframe'
+// TC-## are the qTest test case id's
+/// <reference types="cypress" />
+
+const content = require("../../pages/Content")
+const article = require("../../pages/Article")
+const place = require("../../pages/Place")
+const event = require("../../pages/Event")
+const person = require("../../pages/Person")
+const page = require("../../pages/Page")
+const extend = require("../../pages/Extend")
+const tourPage = require("../../pages/TourPage")
+
+
+describe("Verify the contents of content page", () => {
+
+    //TC-78
+    context("Content page - Contents of the page", () => {
+        it("Verify the contents of Content Page", () => {
+            content.verify()
+        })
+    })
+
+    //TC-79
+    context("Add Content - Click and verify", () => {
+        it("Click on Add content on Content tab from sub-admin tool bar, ", () => {
+            content.clickAndVerify()
+        })
+    })
+
+    //TC-80
+    //Article
+    context("Article - Click and verify", () => {
+        it("Mouse hover on Content tab and navigate to Article link from sub-admin tool bar", () => {
+            article.clickAndVerify()
+        })
+    })
+
+    //Verify and extend required modules
```

Comment:
Comment line should have 1 space before the statement and ended with the fullstop.

Reply:
done